pkg/image/export/main.go

```diff
@@ -392,7 +392,7 @@ done
 
 usage () {
     echo "USAGE: $0 [--image-list rancher-images.txt] [--images rancher-images.tar.gz] --registry my.registry.com:5000"
-    echo "  [-l|--images-list path] text file with list of images. 1 per line."
+    echo "  [-l|--image-list path] text file with list of images. 1 per line."
     echo "  [-l|--images path] tar.gz generated by docker save."
```

Comment:
Is this `-l` going to get fixed as well?

Reply:
different ticket, i hadn't got there. I can do them both in one as i just saw it looking for my next bug